app/components/SettingsCellComponent.js

```diff
@@ -0,0 +1,120 @@
+import React from 'react'
+import {
+  View,
+  Text,
+  StyleSheet,
+  TouchableOpacity,
+  Dimensions,
+  Switch,
+} from 'react-native'
+import { colors } from '../config'
+
+const { settingsCellTextColor, settingsCellBorderLineColor } = colors
+const { width: screenWidth } = Dimensions.get('screen')
+
+const SettingsCellComponent = ({
+  cellText,
+  onCellPress,
+  onSwitchValueChange,
+  isSwitchCellComponent,
+}) => {
+  const cellPressed = () => {
+    if (onCellPress) {
+      onCellPress()
+    }
+  }
```

Comment:
coding choice but we can do this with a ternary operator and it just makes it a bit more concise like

```suggestion
  const cellPressed = onCellPress != null ? onCellPress : null
```

Reply:
<TouchableOpacity
      onPress={cellPressed}
      style={wrapper}
      activeOpacity={cellPressed == null ? 1 : undefined} >
         <Text style={text} numberOfLines={titleNumLines}>{title}</Text>
        {onCellPress != null ? <Switch /> : <Icon name={'chevron'}} />
      <View style={divider} />
    </TouchableOpacity >

for this to be applied i need to wrap this in a view so i can give flexDirection:'row' and that view will be divided in to 2 rows. is it correct @MDHayes ?

Reply:
I think this comment is separate and resolved anyway.